Anlab.Core/Models/OrderDetails.cs

```diff
@@ -23,6 +23,11 @@ public class OrderDetails
 
         public string LabComments { get; set; }
         public decimal AdjustmentAmount { get; set; }
+        public decimal GrandTotal { get

```

Comment:
@jSylvestre But isn't that method only populating the Total?  Once we have adjustment amount we need to add it in, and that's what the grand total is for.  Or am I missing what you are saying here?